src/LargeXlsx/Stylesheet.cs

```diff
@@ -52,6 +52,8 @@ internal class Stylesheet
         private int _nextFillId;
         private int _nextNumberFormatId;
         private int _nextStyleId;
+        private XlsxStyle _lastUsedStyle;
```

Comment:
We could even initialize it as XlsxStyle.Default to achieve the same effect, even if this is not necessary.